core/src/main/java/nl/nn/adapterframework/filesystem/FileSystemSender.java

```diff
@@ -59,7 +60,11 @@
 	@Override
 	public void configure() throws ConfigurationException {
 		super.configure();
-		getFileSystem().configure();
+		FS fileSystem = getFileSystem();
+		if (getApplicationContext()!=null) {
```

Comment:
Wanneer zou de applicationcontext null zijn?

Reply:
in unit tests

Reply:
Dan moet de test aangepast worden, anders hebben we 2 verschillende implementaties